src/lib/components/ui/chat/chat-form.svelte

```
@@ -1,16 +1,18 @@
 <script lang="ts">
 	import { communicateToOllamaModel } from '$lib/services/ollama';
+	import { chatStore, selectedChatStore } from '$lib/store/chat.store';
 	import { Send } from 'lucide-svelte';
 	import { Button } from '$lib/components/ui/button';
 	import ChatAttachment from '$lib/components/ui/chat/chat-attachment.svelte';
 	import ModelDetails from '$lib/components/ui/model-details/model-details.svelte';
 	import { toast } from 'svelte-sonner';
-	import { chatStore, selectedChatStore } from '$lib/store/chat.store';
 	import type { Chat, Message } from '$lib/types';
+	import { IsMobile } from '$lib/hooks/is-mobile.svelte';
```

ESLint

[error] 'IsMobile' is defined but never used.

(@typescript-eslint/no-unused-vars)